docs/README.md

```diff
@@ -0,0 +1,16 @@
+
+## source{d} Ghost Theme Docs
+
+This directory has the assets that are depentents on source{d} Landing, in order to brand-wise comply with the current/updated Landing Identity, [Landing's External Dependents](https://github.com/src-d/landing/blob/master/CONTRIBUTING.md#external-dependents) must be talen into cosideration.
+
+
+### Files
+
+- [Images](https://github.com/src-d/sourced.ghost.io/tree/master/docs/images)
+- [Icons](https://github.com/src-d/sourced.ghost.io/tree/master/docs/icons)
+- [Fonts](https://github.com/src-d/sourced.ghost.io/tree/master/docs/fonts)
```

Comment:
Also I miss info about where are the styles that are applied in ghost, that are responsible for making blog look like landing. Also, if possible I'd mention if those styles are copy/paste from any landing file, or if they were handmade, or copy/paste/modify to satisfy landing design with blog markup.

Reply:
Thanks!